src/Castle.Windsor.Extensions.DependencyInjection/WindsorScopedServiceProvider.cs

```diff
@@ -27,48 +27,40 @@ namespace Castle.Windsor.Extensions.DependencyInjection
 	internal class WindsorScopedServiceProvider : IServiceProvider, ISupportRequiredService, IDisposable
 	{
 		private readonly ExtensionContainerScope scope;
-		private bool disposing = false;
+		private bool disposing;
 
 		private readonly IWindsorContainer container;
 		
 		public WindsorScopedServiceProvider(IWindsorContainer container)
 		{
 			this.container = container;
-			this.scope = ExtensionContainerScope.Current;
+			scope = ExtensionContainerScope.Current;
 		}
 
 		public object GetService(Type serviceType)
 		{
-			using(var fs = new ExtensionContainerScope.ForcedScope(scope))
+			using(var _ = new ForcedScope(scope))
 			{
 				return ResolveInstanceOrNull(serviceType, true);	
 			}
 		}
 
 		public object GetRequiredService(Type serviceType)
 		{
-			using(var fs = new ExtensionContainerScope.ForcedScope(scope))
+			using(var _ = new ForcedScope(scope))
```

Comment:
You can just do `using (new ForcedScope(scope)) { ... }`, no need for the local.

Reply:
chagnes to using(_ = new ForcedScope(scope))
_ is discard, this is correct syntax i beleive